libs/libc/misc/CMakeLists.txt

```diff
@@ -112,4 +112,10 @@ if(CONFIG_FDCHECK)
   list(APPEND SRCS lib_fdcheck.c)
 endif()
 
+add_custom_target(
```

Comment:
Should we check `CONFIG_LIBC_UNAME_DISABLE_TIMESTAMP` before add dependencies?

Reply:
> Should we check `CONFIG_LIBC_UNAME_DISABLE_TIMESTAMP` before add dependencies?

sure, completed